README.md

```diff
@@ -4,17 +4,22 @@
 # What
 This project is developed by all FAC7 cohort from [Founders and Coders](http://www.foundersandcoders.com/)
 
-This is website inspired on Amazon in order to learn React and also apply the other tecnologies we have learn since we started the course. 
+This is website inspired by Amazon in order to learn React and also apply the other technologies we have learned since we started the course.
 
-# How 
-We split on groups of four peopole, each team in on charge on a different component of the webpage. We are going to develop ```React Components```in ordert to re-use them in future progects. 
+# How
+We split on groups of four people, each team in on charge on a different component of the webpage. We are going to develop ```React Components```in order to re-use them in future projects.
 
 # Roles
 
 
 # Technology used
 
+* React.js
+* Hapi.js
+* Node.js
+* Redis
 
+(the RHNR stack? :joy:)

```

Comment:
RRHiNo stack? *R*eact *R*edis *H*ap*i* *No*de. I think we should open an issue about finding the best acronym xD